### ISSUE TITLE
const issue in storage.js

### ISSUE THREAD

**Issue by clibu** [appy-one/acebase#30]
eslint shows the following error _(amongst others)_ in ``storage.js``

````
  1476:29  error    'ret' is constant                                      no-const-assign
````

This also prevents vite -> esbuild from building.

Line 1473 needs to change from ``const ret`` to ``let ret``

eslint shows quite a few errors across multiple source files.

I also noticed that there are no test scripts specified in any package.json and only one test file ``path-info.spec.js``
Does AceBase have a comprehensive test suite? If not is this planned.

**Comment by appy-one** [comment]
Good catch. I've made quite a few changes to the code to enable clustering in nodejs environments, I've now also changed the line you referred to, hope to be able to commit them today. Regarding your test question - I have many test files locally that I run before publishing updates, but they are not suitable for automated testing atm

**Comment by clibu** [comment]
@appy-one Any progress on the new release yet, not that I'm in a hurry.

Re. the tests are you planning on automating and publishing them at some point?

**Comment by appy-one** [comment]
Fix published in v1.6.0. Don't know when I'll be working on automated tests
